bitnami/fluentd/README.md

```diff
@@ -326,6 +367,10 @@ aggregator.extraEnv[1].name=ELASTICSEARCH_PORT
 aggregator.extraEnv[1].value=your-port-here
 ```
 
+#### By changing the value at `aggregator.extraConfig`
+
+An example of how to do this is available in the `values-production.yaml`
```

Comment:
We are considering removing the `values-production.yaml` file soon, so this would not be applicable anymore. Could you copy the example here instead?

Reply:
Ok.